<?xml version="1.0"?>
<launch>

  <arg name="hostname" default="localhost" />

  <arg name="save_dir" default="$(find squirrel_active_exploration)/data" />
  <arg name="cloud_name" default="$(find squirrel_segmentation)/data/test45.pcd" />
  <arg name="saliency_name" default="$(find squirrel_segmentation)/data/test45_mask_complete.png" />

  <node name="squirrel_active_exploration"
        pkg="squirrel_active_exploration"
        type="squirrel_active_exploration"
        output="screen">
        <param name="save_dir" type="string" value="$(arg save_dir)"/> 
        <param name="cloud_name" type="string" value="$(arg cloud_name)"/>
        <param name="saliency_name" type="string" value="$(arg saliency_name)"/> 
  </node>


</launch>
